src/index.js

```diff
@@ -8,13 +8,7 @@ const mergeFolders = require('./merge-folders').merge
 function buildBundle (config) {
   return rollup.rollup(config)
     .then(function (bundle) {
-      return bundle.write({
-        format: config.format || 'es6',
```

Comment:
You are using "rollup.config". Then "bundle" variable came from Rollup. Then "bundle" makes a new output.

Thus could you explain why you need any "logic" in the middle?